src/components/ArticlesList/ArticlesList.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../../api";
+import ArticleCard from "../ArticleCard.jsx/ArticleCard";
+import "./ArticlesList.css"
+
+const ArticlesList = () => {
+
+    const [articlesList, setArticlesList] = useState([])
+
+    useEffect(() => {
+        getArticles().then(({ articles }) => {
+            setArticlesList(articles)
+        })
+    }, [])
+
+    return (
```

Comment:
We should take into account that the data might take some time to be fetched, and so we should have a loading screen to give feedback to the user 🙂